lambdas/backend/express-route-handlers.js

```diff
@@ -249,6 +252,42 @@ function putMarketplaceSubscription(req, res) {
     })
 }
 
+function getFeedback(req, res) {
+    function error(data) {
+        console.log(`error: ${data}`)
+        res.status(500).json(data)
+    }
+
+    function success(data) {
+        res.status(200).json(data)
+    }
+
+    if (!feedbackEnabled) {
+        error("Customer feedback not enabled")
+    } else {
+        feedbackController.fetchFeedback(error, success)
```

Comment:
This could be refactored to return a promise. The JS SDKs all offer a: sdk.operation(args).promise() method that returns a promise.

Reply:
Dunzo